README.md

```diff
@@ -56,23 +56,39 @@ To manually install the connector on a local installation of Confluent:
 
 ### Docker
 
-There is a [`docker-compose.yml`](docker-compose.yml) file included in this repository that can be used to run the connector locally using [Docker Compose](https://docs.docker.com/compose/). The Docker Compose file is based on the [Confluent Platform Docker images](https://docs.confluent.io/platform/current/installation/docker/image-reference.html).
+There is a [`docker-compose.yml`](docker-compose.yml) file for standalone mode and an alternative [`docker-compose-distributed.yml`](docker-compose-distributed.yml) for distributed mode included in this repository that can be used to run the connector locally using [Docker Compose](https://docs.docker.com/compose/). The Docker Compose file is based on the [Confluent Platform Docker images](https://docs.confluent.io/platform/current/installation/docker/image-reference.html).
 
 1. Create and configure a [configuration file](#configuration) ensuring you have set at least the basic properties.
+**Note:** You must provide connector properties when starting connector in distributed mode. 
+An example cURL command to start the connector in distributed mode is:
 
+
+```shell
+ curl -X POST -H "Content-Type: application/json" --data '{"name": "ably-channel-sink",
+     "config": {"connector.class":"com.ably.kafka.connect.ChannelSinkConnector", "tasks.max":"3", 
+     "group.id":"ably-connect-cluster",
+     "topics":"topic1,topic2","client.id":"Ably-Kafka-Connector","channel":"#{topic}","message.name": "#{topic}_message",
+     "client.key":"<Put your API key here>" }}' http://localhost:8083/connectors
+```
 2. Start the cluster using:
 
     `docker-compose up -d`
+ for standalone mode or
+
+   `docker-compose -f docker-compose-distributed.yml up -d` for distributed mode.
+   
+**Note**: You can view the logs using `docker-compose logs connector`
+
+**Note 2**: You must start your connectors using [Connect REST interface](https://docs.confluent.io/platform/current/connect/references/restapi.html) when using distributed mode.
 
-    **Note**: You can view the logs using `docker-compose logs connector`
 
-3. Once the containers have started, you can test the connector by subscribing to your Ably channel using [SSE](https://ably.com/documentation/sse) in a new terminal window. Replace `<channel-name>` with the channel set in your configuration file and `<ably-api-key>` with an API key with the capability to subscribe to the channel.
+4. Once the containers have started, you can test the connector by subscribing to your Ably channel using [SSE](https://ably.com/documentation/sse) in a new terminal window. Replace `<channel-name>` with the channel set in your configuration file and `<ably-api-key>` with an API key with the capability to subscribe to the channel.
```

Comment:
Why has the numbering changed and now there's `4` after `2`? :thinking:

Reply:
Oops, that must have happened automatically by entering. Fix is 37661743b5f394e4c239c269ccae938d0e3791e2